pkgs/stdenv/cross/default.nix

```diff
@@ -68,7 +68,7 @@ in lib.init bootStages ++ [
            else if crossSystem.useLLVM or false
              then buildPackages.llvmPackages_8.lldClang
            else if crossSystem.isZ80
-             then buildPackages.z88dk
+             then buildPackages.z88dk-unwrapped
```

Comment:
This should be `buildPackages.z88dk` so I guess we'll need to figure out a different way to break the cycle.